tests/tests/time/time list render large list.coffee

```diff
@@ -1,6 +1,13 @@
 require('../../testutils.js').plug_macros()
 
 module.exports =
+  # MWF MACHINE
+  #
+  # Firefox: 3908 -> 4131
+  # Chrome : 2496 -> 2597
+  # Ie     :   -1 ->   -1

```

Comment:
Der er pt. en fejl der gør at når listen er stor nok,
så bliver IE forvirret og lukker den med det samme igen.

Se "fails/Should not fire leave event on button click (ie)"